zcash_client_sqlite/src/wallet/init/migrations/add_transaction_views.rs

```diff
@@ -225,15 +252,22 @@ impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
                        transactions.raw              AS raw,
                        transactions.fee              AS fee,
                        -sent_notes.value             AS value,
-                       false                         AS is_change,
+                       CASE
+                           WHEN sent_notes.from_account = sent_notes.to_account THEN 1
+                           ELSE 0
+                       END AS is_wallet_internal,
+                       0                             AS is_change,
+                       0                             AS received_count,
+                       1                             AS sent_count,
```

Comment:
Now that the `sent_notes` table is also storing change / wallet-internal outputs, this is going to count these outputs as part of `sent_note_count`. But I presumed that `sent_note_count` is intended to be a count of the number of logical (e.g. from a payment request) recipients in the transaction? If that's the case, then this could be replaced by `is_not_wallet_internal` above (or separately count the number of change / wallet-internal notes).

Reply:
Actually, we can just change the definition of `sent_count` and fix both these issues at once.